README.md

```diff
@@ -0,0 +1,24 @@
+# GOV.UK Repository Mirrorer
+
+Ensures all repositories tagged with "govuk" in the alphagov organisation in Github.com are mirrored to Gitlab.com.
+
+We use Gitlab.com so we can still deploy in the event that Github.com is down. We also use the private repositories at Gitlab in order to fix any security vulnerabilities without disclosing them.
+
+This script will ensure a project exists in the `govuk` namespace in Gitlab.com for every `govuk` Github repository. It will then synchronise all branches and tags, including removing branches that have been deleted in Github. It is designed to be run periodically in Jenkins.
+
+**Warning:** If you need to work on a branch in private on Gitlab.com, this script must be prevented from running. If it runs and the branch you're working on does not exist in Github, it will be removed from Gitlab.
+
+## Usage
+
+Required environment variables:
+
+| Variable                   | Description                                                                                                         |
+|:---------------------------|:--------------------------------------------------------------------------------------------------------------------|
+| `GITHUB_ACCESS_TOKEN`      | A personal access token with `read:org` and `repo` scope                                                            |
+| `GITLAB_API_ENDPOINT`      | The URL for the Gitlab API, probably `https://gitlab.com/api/v4`                                                    |
```

Comment:
I think `GITLAB_API_ENDPOINT` isn't used in the code?

Reply:
It's part of the [`gitlab` gem](https://github.com/NARKOZ/gitlab/search?q=GITLAB_API_ENDPOINT).